### PR TITLE
Update debug_client_main.cpp

### PR DESCRIPTION
**Pull request by lost0427**
Dear Neboer,

I trust this message finds you in good spirits.

I wish to convey that I have updated the code in accordance with the esteemed directives set forth in `src/cli/readme.md`. Yet, I find myself pondering if this endeavor aligns with your illustrious expectations. Might I kindly beseech your confirmation on whether my efforts are indeed in harmony with the guidelines?

Furthermore, I am most eager to comprehend the rationale behind the migration of the code to `debug_client_main.cpp`. Your esteemed insights on this matter would be of immense value.

I extend my heartfelt gratitude for your unwavering support and guidance.

Retrieve0.

### CODE REVIEW

**Comment by peigongdsd** [comment]
Just go fuck yourself and stick to some human readable bullshit, okay?😅

File: src/cli/debug_client_main.cpp [hunk 1]
```diff
@@ -1,20 +1,50 @@
...
-    std::cin >> request_content;
+    if (argc < 3) {
+        std::cerr << "Usage: " << argv[0] << " <command> <argument>" << std::endl;
+        return 1;
+    }
+
+    std::string p1 = argv[1];
+    std::string p2 = argv[2];
+
+    nlohmann::json request_content = cli(p1, p2); // Get the request content from cli
```
Comment (Neboer) [inline src/cli/debug_client_main.cpp:41]: 混用nlohmann::json与json

**Comment by lost0427** [comment]
Dear  neboer,

I hope you are doing well.

I have changed the code according to `src/cli/readme.md`. However, I am not sure if I did it right. Could you please let me know?

Also, could you explain why the code was moved to `debug_client_main.cpp`?

Thank you for your help!

Retrieve0

**Comment by lost0427** [comment]
①你看看这是不是想要的方式（nmm ep path ）然后自动生成JSON传给后端 ②为什么main的代码全去debug_client_main了 为了调试？ ③我谢谢你

**Pull request closed without merging**